Cannot compile example of I2S driver for the ESP32-S3

I was already running original [LED driver from hpwit](https://github.com/hpwit/I2SClockLessLedDriveresp32s3). 
I tried the insert version directly into FastLED strictly by example, but I can't compile:

```.ino:61:undefined reference to `_ZN2fl17InternalI2SDriver6createEv'```

--- a/src/platforms/esp/32/clockless_i2s_esp32s3.cpp
+++ b/src/platforms/esp/32/clockless_i2s_esp32s3.cpp
@@ -1,14 +1,9 @@
-
-
-
-#if defined(CONFIG_IDF_TARGET_ESP32S3) || defined(ARDUINO_XIAO_ESP32S3)
-
 #define FASTLED_INTERNAL
-#include "third_party/yves/I2SClockLessLedDriveresp32s3/driver.h"
 #include "FastLED.h"
 
+#if defined(CONFIG_IDF_TARGET_ESP32S3)
 
-
+#include "third_party/yves/I2SClockLessLedDriveresp32s3/driver.h"
 
 #include "crgb.h"
 #include "eorder.h"

